lib/spark_api/models/contact.rb

```diff
@@ -45,6 +45,10 @@ def listing_carts(arguments = {})
         @listing_carts ||= ListingCart.collect(connection.get("/contacts/#{self.Id}/listingcarts", arguments))
       end
 
+      def messages(arguments = {})
+        @messages ||= Message.collect(connection.get("/contacts/#{self.Id}/messages", arguments))
```

Comment:
Don't merge this just yet. After talking with @joshcom I realized that this method is a little misleading and should be removed.